src/hmem_cuda.c

```diff
@@ -744,6 +751,11 @@ int cuda_host_unregister(void *ptr)
 	return -FI_EIO;
 }
 
+enum cuda_xfer_setting cuda_get_xfer_setting()
```

Comment:
nit: Add `void` inside the parameter list of the function signature

Reply:
fixed in new revision